tck/old-tck-selenium/commandLink/src/main/webapp/WEB-INF/faces-config.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-    Copyright (c) 2009, 2018 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2005, 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
2005 is the correct year: https://github.com/jakartaee/faces/blob/3fae98234692ec16545a6d27cf36fabaeb883f9b/tck/old-tck/source/src/com/sun/ts/tests/jsf/spec/render/commandlink/faces-config.xml#L4